### PR TITLE
Implement staff screen design

### LINKED ISSUE #358: Implement Staff screen design
## Kind (Required)

- Improvement

## Overview (Required)

- We don't have a design. But probably we can use contributor design or speaker design 😄 
You can access staffs screen  from about screen

 ## Links	

-

### PR DESCRIPTION
**Pull request by minami1389**
## Issue
- close #358 

## Overview (Required)
- I implemented staff screen design with reference to contributors design.
- While loading icon images, the gray background color is displayed.
- To use Context in StaffItem, I added `fun appComponent(): Context` in AppComponent and StaffComponent.
  - Is this the right way?

## Links
-

## Screenshot
Before | After
:--: | :--:
 | 

## Movie
Jan-21-2020 00-01-33

### CODE REVIEW

**Comment by takahirom** [comment]
Thank you for the contribution!
For your reference, If you add an email to commit. You can leave your commit in contributors.  
https://github.com/DroidKaigi/conference-app-2020/graphs/contributors
git config --global user.email your_github_registered_email@example.com

File: feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/item/StaffItem.kt [hunk 1]
```diff
@@ -9,13 +15,29 @@ import io.github.droidkaigi.confsched2020.staff.R
...
 
 class StaffItem @AssistedInject constructor(
-    @Assisted private val staff: Staff
+    @Assisted private val staff: Staff,
+    context: Context,
+    private val lifecycleOwnerLiveData: LiveData<LifecycleOwner>
 ) : BindableItem<ItemStaffBinding>(staff.id.hashCode().toLong()),
     EqualableContentsProvider {
+
+    private val placeHolder = VectorDrawableCompat.create(
```
Comment (takahirom) [inline feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/item/StaffItem.kt]: Thank you!
We can use lazyWithParam like this. So you don't have to context for the constructor 👍 

```kotlin
    private val layoutInflater by lazyWithParam<Context, LayoutInflater> { context ->
        LayoutInflater.from(context)
    }
```

**Comment by takahirom** [comment]
Can you merge from master?

**Comment by yurihondo** [comment]
👀

**Comment by minami1389** [comment]
Sorry for the late reply.
I committed as below.

- merged from master.
- get from context from view, no longer have context in constructor.
  - Am I getting context properly?
  - I didn't know how to use `lazyWithParam`. How should I use it...?

**Pull request merged**